src/integration/telegram/hybrid_native_bot.py

```
@@ -1,47 +1,146 @@
+    async def connect(self, websocket: WebSocket):
+        await websocket.accept()
+        self.active_connections.append(websocket)
+
+    def disconnect(self, websocket: WebSocket):
+        self.active_connections.remove(websocket)
+    
+    async def broadcast_proactive_message(self, message: dict):
+        for connection in self.active_connections:
+            try:
+                await connection.send_text(json.dumps(message))
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)